src/Sdk/DTPipelines/workflow-v1.0.json

```diff
@@ -381,7 +381,7 @@
     "container-mapping": {
       "mapping": {
         "properties": {
-          "image": "non-empty-string",
```

Comment:
This allows `services`'s `container.image` to be empty, can the code where we start the services handle that?